src/components/SearchBar/SearchBar.js

```diff
@@ -34,12 +34,12 @@ export class SearchBar extends React.Component {
 
 
         return (

```

Comment:
this comment is not in place, i mean my comment : ) and its not related to this commit which deals with BEM
I was looking at the entire file `SearchBar.js` and wanted to say that **event handlers** , meaning `handleTermChange` , `searchWithEnter` and `search`. all those methods of the Class, are called **event handlers**, they should be written as arrow functions

```
searchWithEnter = (event) => {
    if(event.keyCode === 13){
        this.search();
     }
}
```

An arrow function expression is a syntactically compact alternative to a regular function expression, although without its own bindings to the **this**. It is best suited for handlers since they are attached to piece of code not in your control such as `onChange` and you are loosing the `this`. And that is why you bind it in **constructor**. But if you use arrow functions, you dont need to bind it in the constructor, less code to write.

Reply:
I didn't quite get what you ment. how come I don't need to use `this` when using an arrow function?
Is it "automatically" bound to the component?